### PR TITLE
Fixes 112: Add #pkgs to the repository table

### PR DESCRIPTION
**Pull request by avisiedo**
- Update ContentItem interface to add package_count field.
- Update view to add the column with the package_count field.
- The column title should be `Packages`.
- Its position will be after `Version` column.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-112

File: src/components/ContentListTable/ContentListTable.tsx [hunk 1]
```diff
@@ -210,11 +210,12 @@ const ContentListTable = () => {
             </Thead>
             <Tbody>
               {contentList.map(
-                ({ uuid, name, url, distribution_arch, distribution_versions }: ContentItem) => (
+                ({ uuid, name, url, distribution_arch, package_count, distribution_versions }: ContentItem) => (
```
Comment (Andrewgdewar) [inline src/components/ContentListTable/ContentListTable.tsx:213]: What value will you display if it is zero? 
"Unknown", 0, "N/A" ? 
The below is the screenshot, I think we may need design input here though. 
We'll likely want to have some N/A test and a hover popup that explains why.
Screen Shot 2022-09-06 at 9 02 40 AM

File: src/components/AddContent/helpers.test.ts [hunk 2]
```diff
@@ -34,6 +34,7 @@ it('mapFormikToAPIValues', () => {
       distribution_arch: 'x86_64',
       distribution_versions: ['el7'],
       gpgKey: '',
+      package_count: 0,
```
Comment (Andrewgdewar) [inline src/components/AddContent/helpers.test.ts:37]: not needed here

File: src/components/AddContent/helpers.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ export const mapFormikToAPIValues = (
     name,
     url,
     distribution_arch: arch,
+    package_count: 0,
```
Comment (Andrewgdewar) [inline src/components/AddContent/helpers.ts:29]: not needed here.

File: src/components/ContentListTable/ContentListTable.tsx (hunk 1 above)
Comment (Andrewgdewar) [reply src/components/ContentListTable/ContentListTable.tsx:213]: For now maybe leave it as "0" and we can address what do here at a later time.

**Review by Andrewgdewar — APPROVED** [review]
Ack!

**Pull request merged**